### DISCUSSION TITLE
IME

### DISCUSSION THREAD

**Discussion by Drodofsky** [pop-os/libcosmic#535] [General]
Hello,

It would be great if libcosmic could support IME (Input Method Editor).
There has been a patch enabling IME for iced for a long time, but it has not yet been merged. At the very least, please enable IME on the winit side, so that users can utilize cosmic applications. While it may not look perfect, it will at least be functional.

**Comment by mmstick** [comment]
We will rely on support in iced, so that would be the best place to request it.
